multicluster/apis/multicluster/v1alpha1/clusterset_types.go

```diff
@@ -73,11 +75,11 @@ type ClusterConditionType string
 const (
 	// ClusterReady indicates whether Cluster is ready and connected.
 	ClusterReady ClusterConditionType = "Ready"
-	// ClusterIsElectedLeader indicates whether the leader cluster is the elected leader.
+	// ClusterIsLeader indicates whether the leader cluster is the right leader.
 	// Used in Member clusters only.
-	ClusterIsElectedLeader ClusterConditionType = "IsElectedLeader"
-	// ClusterImportsResources indicates whether the member cluster has elected the
-	// local cluster as the elected leader and imports resources from it.
+	ClusterIsLeader ClusterConditionType = "IsLeader"
+	// ClusterImportsResources indicates whether the member cluster has connected to the
+	// local cluster as the leader and imports resources from it.
```

Comment:
Maybe remove "and imports resources from it".

Reply:
Done